user_tools/tests/spark_rapids_tools_e2e/features/event_log_processing.feature

```diff
@@ -41,3 +41,16 @@ Feature: Event Log Processing
       Qualification. Raised an error in phase [Execution]
       """
     And return code is "1"
+
+  @test_id_ELP_0003
+  Scenario Outline: Qualification tool processes event logs with different execution engine
```

Comment:
nit: `execution engines`

Q: what is the behavior if a user input `--platform onprem`, but with a photon event log? Does this fail in python side early?

Reply:
After discussing with @mattahrens and @leewyang, we have decided to fallback to the default runtime for the platform (e.g., `SPARK` for `onprem`).

Filed a follow up issue for this case #1420